django_lightweight_queue/middleware/transaction.py

```diff
@@ -1,14 +1,29 @@
 from django.db import transaction, connection
 
+from ..task import task
+
 class TransactionMiddleware(object):
+    _transaction_started = False
```

Comment:
It's Better Practice™ to set these in `__init__` than share the class-level instance.